File: Libs/cs-script/Samples/hello.cpp
#pragma comment(lib, "User32.lib")
//css_ref System.Windows.Forms;

#include "windows.h"
using namespace System;
using namespace System::Windows::Forms;

public ref class Script
{
	public: 
		static void Main()
		{
			Console::WriteLine(L"Hello World! (C++)");
			MessageBoxA(0, "Hello World! (C++)", "Non-Managed", 0);
			MessageBox::Show("Hello World! (C++)", "Managed");
		} 
};

File: Libs/cs-script/Samples/hello.cs
using System;
using System.Windows.Forms; 

class Script
{ 
	const string usage = "Usage: cscscript hello ...\nThe canonical \"Hello world!\" script.\n";

	static public void Main(string[] args)
	{
		if (args.Length == 1 && (args[0] == "?" || args[0] == "/?" || args[0] == "-?" || args[0].ToLower() == "help"))
		{
			Console.WriteLine(usage);
		}
		else
		{
			Console.WriteLine( "Hello World!" ); 
			MessageBox.Show( "Hello World!"); 
		}
	}
}
